features/deputy.feature

```diff
@@ -106,6 +106,16 @@ Feature:
               "target_message": "Content of a deputy message",
               "target_firstname": "D\u00e9put\u00e9"
           }
+      },
+      {
+        "Email": "deputy-ch-li@en-marche-dev.fr",
+        "Name": "D\u00e9put\u00e9 CHLI FDESIX",
```

Comment:
just fyi (no need to change anything here), thanks to PHPMatcher we now use in the `JsonContext`, we can use special characters in these tests.
you could write `"Name": "Député CHLI FDESIX"` here for example.
i think it makes tests more readable.